rust/kernel/str.rs

```diff
@@ -19,7 +19,9 @@ pub type BStr = [u8];
 ///
 /// # Examples
 ///
-/// ```rust,no_run
+/// ```no_run
```

Comment:
I was not sure whether to remove this -- we do not gain much by running this kind of example, but on the other hand avoiding clutter looks better and it would mean when we actually use `no_run` it is because it cannot be run for some reason.